frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IParcel } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import './InfoSlideOut.scss';
+import { formatMoney } from 'utils/numberFormatUtils';
+
+interface IParcelAttributes {
+  /** the selected parcel information */
+  parcelInfo: IParcel;
+}
+
+/**
+ * Displays parcel specific information needed on the information slide out
+ * @param parcelInfo the selected parcel data
+ */
+export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo }) => {
+  let formatAssessed;
+  if (parcelInfo?.assessed) {
+    formatAssessed = formatMoney(parcelInfo?.assessed);
+  } else if (parcelInfo?.evaluations[0].value) {
```

Comment:
I think this will throw an exception if parcelInfo.assessed is undefined and parcelInfo.evaluations is defined but is equal to [].

Reply:
is this where I would use getIn()?